Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -77,6 +77,112 @@ export function updateDeploymentStatus(publishingProfile, isDeploymentSuccess: b
 }
 
 /**
+ * Get all continious web jobs 
+ * 
+ * @param   publishingProfile     Publish Profile details
+ * 
+ * @returns promise with string
+ */
+export function getAllContinuousWebJobs(publishingProfile): Q.Promise<string>  {
+    var deferred = Q.defer<string>();
+
+    var webAppPublishKuduUrl = publishingProfile.publishUrl;
+    tl.debug('Web App Publish Kudu URL: ' + webAppPublishKuduUrl);
+    if(webAppPublishKuduUrl) {
+        var requestUrl = "https://" + webAppPublishKuduUrl + "/api/continuouswebjobs"
+        var accessToken = 'Basic ' + (new Buffer(publishingProfile.userName + ':' + publishingProfile.userPWD).toString('base64'));
+        var headers = {
+            authorization: accessToken
+        };
+
+        httpObj.get('GET', requestUrl, headers,
```

Comment:
@arjgupta - can you review for compliance with the proxy changes that you are doing.